.travis.yml

```diff
@@ -318,4 +323,6 @@ install:
       export -f run_tests
 
 script:
+    # must be run too:
```

Comment:
I don't know how to to that

Reply:
Note: for https://github.com/symfony/symfony/pull/34554/commits/3a7a330c3656f363e32691008ff295fe2c7f0323 tests must be red. So it does not work